### PR TITLE
Fix inconsistencies and invalid URL on ti setup check

### PR DESCRIPTION
**Pull request by ewanharris**
https://jira.appcelerator.org/browse/TIMOB-26206
https://jira.appcelerator.org/browse/TIMOB-24892

- If process.mainModule.filename points to the appc bin then try and resolve titanium so we list the correct node modules
- Replace preview.appcelerator.com with preview.appcelerator.com/appc-studio to fix the 502

### CODE REVIEW

File: lib/commands/setup.js [hunk 1]
```diff
@@ -669,6 +669,10 @@ SetupScreens.prototype.check = function check(callback) {
 				var cwd = process.mainModule && process.mainModule.filename,
 					root = path.resolve('/');
 
+				if (path.basename(cwd) === 'appc') {
+					cwd = require.resolve('titanium');
```
Comment (cb1kenobi) [inline lib/commands/setup.js]: Why can't you just do `cwd = path.resolve(__dirname, '..', '..');`?

Comment (ewanharris) [reply lib/commands/setup.js]: That makes more sense, and removes the need to even have this check. Updating

**Comment by cb1kenobi** [comment]
**APPROVED**

**Comment by hansemannn** [comment]
@ewanharris Was this published so far? Trying with 5.1.1 and still see this issue.

**Comment by cb1kenobi** [comment]
This fix has not been published yet.

**Pull request merged**